manager/config/prod/deployment_configmap.yaml

```diff
@@ -42,3 +49,8 @@ data:
   #external vault  variables
   USER_VAULT_ADDRESS: "http://vault.m4d-system.svc.cluster.local:8200/"
   USER_VAULT_PATH: "external"
+  APPID: "datauser1/notebook-with-kafka"
```

Comment:
what is it?

Reply:
In a normal flow, APPID is passed to connectors from pilot (https://github.com/IBM/the-mesh-for-data/blob/master/pkg/connectors/protos/data_catalog_request.proto (2) https://github.com/IBM/the-mesh-for-data/blob/master/pkg/connectors/protos/data_credential_request.proto (3) https://github.com/IBM/the-mesh-for-data/blob/acb9a2c96963ac263a3867431b56609a02bc688b/pkg/connectors/protos/policy_manager_request.proto#L36). 

Since we are trying to mock pilot here, I needed a place where the APPID could be defined for creating a mock pilot requests. Hence I used this variable in the deployment_configmap.yaml. It could have been defined in the files related to mockup code as well (which leads to duplication of the configuration). Please let me know if I need to move it to files related to mockup code i.e. in the following locations:

(1) https://github.com/rohithdv/the-mesh-for-data/blob/baa89451b9b901ddbfe3348e72c404a0e2f8198b/test/services/pilot/catalog-credential-mock/mockup_pilot_catalog_con.go#L62 

(2) https://github.com/rohithdv/the-mesh-for-data/blob/baa89451b9b901ddbfe3348e72c404a0e2f8198b/test/services/pilot/policy-manager-mock/mockup_pilot.go#L89 ?)

Reply:
Yes, it shouldn't be part of the deployment configmap.